modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc

```diff
@@ -66,7 +71,7 @@ spec:
   results:
     - description: CVE result format  # <4>
       name: SCAN_OUTPUT
-      type: string
+      value: string
```

Comment:
1. `type` is the correct key
2. string is default value so we can omit this line completely

Reply:
@ppitonak Updated. Could you please review and add LGTM or if you have anyother feedback please let me know?